herpetological-conservation-and-biology.csl

```diff
@@ -0,0 +1,229 @@
+                      <number text-case="capitalize-first" variable="edition" form="ordinal"/>
+                      <text term="edition" text-case="capitalize-first" suffix=". "/>
+                    </group>
+                  </if>
+                  <else>
+                    <text variable="edition"/>
+                    <number text-case="capitalize-first" variable="edition" form="ordinal"/>
+                    <text term="edition" text-case="capitalize-first" suffix=". "/>
+                  </else>
+                </choose>
+              </if>
+            </choose>
+            <group delimiter=" ">
+              <text variable="collection-title" text-case="capitalize-all"/>
+              <text variable="number"/>
+            </group>
+          </if>
+        </choose>
+      </else-if>
+      <else-if type="chapter paper-conference" match="none">
+        <text variable="container-title"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <choose>
+      <if type="article-journal article-magazine" variable="collection-title" match="none">
+        <group delimiter=", ">
+          <group delimiter=" ">
+            <text variable="collection-title"/>
+            <text variable="collection-number"/>
+          </group>
+          <text variable="genre"/>
+          <text variable="publisher"/>
+          <text variable="publisher-place"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if type="report" match="none">
+        <choose>
+          <if is-numeric="edition">
+            <group delimiter=" ">
+              <number text-case="capitalize-first" variable="edition" form="ordinal"/>
+              <text term="edition" text-case="capitalize-first"/>
+            </group>
+          </if>
+          <else>
+            <text variable="edition"/>
+            <number text-case="capitalize-first" variable="edition" form="ordinal"/>
+            <text term="edition" text-case="capitalize-first"/>
+          </else>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="volume">
+    <choose>
+      <if type="article-journal article-magazine" match="none">
+        <choose>
+          <if variable="volume">
+            <group delimiter=" ">
+              <text term="volume" text-case="capitalize-first"/>
+              <text variable="volume"/>
+            </group>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <choose>
+      <if type="thesis report" match="any">
+        <text variable="number-of-pages" suffix=" p."/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="webpage">
+    <choose>
+      <if type="webpage">
+        <text variable="number" suffix=". "/>
+        <text variable="URL" prefix="Available from " suffix=". "/>
+        <group prefix="[" suffix="]">
+          <text value="accessed" text-case="capitalize-first" suffix=" "/>
```

Comment:
I meant this one: `<text term="accessed" text-case="capitalize-first" suffix=" "/>` should do exactly the same (and is more robust for future changes and localization to other languages)